src/applications/proxy-rewrite/tests/redirects/index.unit.spec.js

```diff
@@ -29,3 +30,27 @@ describe('Redirect replaced pages', () => {
     expect(fakeWindow.location.href).to.be.undefined;
   });
 });
+
+describe('Validate crossDomainRedirects.json', () => {
+  const redirectsBySource = redirects.reduce((grouped, redirect) => {
+    const fullPath = `https://${redirect.domain}${redirect.src}`;
+    const items = grouped[fullPath] || [];
+    return {
+      ...grouped,
+      [fullPath]: items.concat(redirect.dest),
+    };
+  }, {});
+
+  Object.entries(redirectsBySource).forEach(([fullSource, destinations]) => {
+    it(`${fullSource} is a valid URL`, () => {
+      const url = new URL(fullSource);
+      expect(fullSource).to.be.equal(url.href);
+    });
+
+    it(`${fullSource} has a unique valid destination`, () => {
+      expect(destinations.length).to.be.equal(1);
+      const destUrl = new URL(destinations[0], 'https://www.va.gov');
+      expect(destinations[0]).to.be.equal(destUrl.pathname);
+    });
+  });
+});
```

Comment:
This validation mostly prevent typos going into the redirects file, as sometimes we do a bunch in bulk so it's easy to miss something.